frontend/src/app/shared/services/helper/user.service.ts

```diff
@@ -46,14 +46,22 @@ export class UserService implements IUserService {
     }
   }
 
+  getAllActiveUsers$(): Observable<User[]> {
+    return this.getAllUsers$().pipe(map(users => users.filter(user => user.active)));
+  }
+
   updateUserCache(): void {
     if (!this.users$) {
       this.users$ = new BehaviorSubject<User[]>([]);
     }
+    forkJoin({
+      activeUsers: this.userApiService.getActiveUsers$(),
+      users: this.userApiService.getUsers$()
+    }).pipe(map (({ activeUsers, users}) => {
+        users.forEach(user => user.active = !!activeUsers.find(activeUser => activeUser.id === user.id));
 
-    this.userApiService
-      .getUsers$()
-      .pipe(take(1))
+        return users;
+    }))
       .subscribe(u => this.users$.next(u));
```

Comment:
Hier fehlt noch das take(1) (damit das Obeservable beendet wird)
Kann in die Pipe hinter map(...) geschrieben werden (ca. Zeile 64)